bifrost.go

```diff
@@ -85,6 +88,28 @@ func NewRainbowBridge(bc *BridgeConfig) (RainbowBridge, error) {
 	}
 
 }
+func newPinataIPFS(g *BridgeConfig) (rainbowBridge, error) {
```

Comment:
Can we add spacing between these two....and also comment as necessary? Basically, let's try to follow the existing code pattern 🙏